### ISSUE TITLE
Get Device colors palette 

### ISSUE THREAD

**Issue by FouadMagdy01** [callstack/react-native-paper#3950]

**Comment by lukewalczak** [comment]
Hey @FouadMagdy01, firstly run `yarn bootstrap` to install all required modules, then go to the example directory and run `yarn android:native` to install the special version of Expo Go that includes the native module.

**Comment by FouadMagdy01** [comment]
> Hey @FouadMagdy01, firstly run `yarn bootstrap` to install all required modules, then go to the example directory and run `yarn android:native` to install the special version of Expo Go that includes the native module.

![Screenshot_4](https://github.com/callstack/react-native-paper/assets/88867595/5c131c23-32ab-43f5-98d0-a5513dc6a6fa)

look at this
the yarn command literally do nothing in the cloned file
it works in all system directories but here it doesnt make any effect

and what about my first question? how to get the device color pallete?

**Comment by lukewalczak** [comment]
> look at this
the yarn command literally do nothing in the cloned file

What do you mean by the cloned file? Did you cloned/fork the `react-native-paper` repository?

> how to get the device color palette?

We're using that library: https://github.com/pchmn/expo-material3-theme

**Comment by FouadMagdy01** [comment]
> > look at this
> > the yarn command literally do nothing in the cloned file
> 
> What do you mean by the cloned file? Did you cloned/fork the `react-native-paper` repository?
> 
> > how to get the device color palette?
> 
> We're using that library: https://github.com/pchmn/expo-material3-theme

Yes i cloned the main repo by using git clone and the url of the main repo

Then i cant run it on my emulator or physical device

All yarn commands do nothing in the cloned file
I don't know if the issue is in my system or no

But i double checked that all env vars are correctly configured

The weird thing is that the commands are working fine in every directory in my system

I also tried to download the code as a zip file and copied all folders to a newly created folder but it didn't work also


i tried to create a new empty folder and tested yarn command and it worked 

when i copied  the downloaded files the yarn command dont do anything

look at this pic

![Screenshot_4](https://github.com/callstack/react-native-paper/assets/88867595/f95b252d-d218-49e0-8b30-b5b5e23b064c)

the first command is before copying files

the second is after copying

and this is the folder structure after copying files

![Screenshot_6](https://github.com/callstack/react-native-paper/assets/88867595/ec21f179-958e-4cef-8f7d-a4c6e299ffca)

Its very weird honestly

**Comment by FouadMagdy01** [comment]
sorry, i posted a wrong image, i editted the answer and added the correct screenshot

do u have any solutions ?

**Comment by lukewalczak** [comment]
Hey @FouadMagdy01, it looks like there is an issue with your local setup, make sure that your environment is properly set.

**Comment by FouadMagdy01** [comment]
> Hey @FouadMagdy01, it looks like there is an issue with your local setup, make sure that your environment is properly set.


I configured them well
But no problem
If you found any solution for this problem contact me
Thank you
